pkg/controller/account/account_controller.go

```diff
@@ -706,6 +709,56 @@ func matchSubstring(roleID, role string) (bool, error) {
 	return matched, err
 }
 
+// getConfigDataByKey gets the desired key from the default configmap or returns an empty string and error
+func getConfigDataByKey(kubeClient client.Client, key string) (string, error) {
+	configMap := &corev1.ConfigMap{}
+	err := kubeClient.Get(
+		context.TODO(),
+		types.NamespacedName{Namespace: awsv1alpha1.AccountCrNamespace,
+			Name: awsv1alpha1.DefaultConfigMap}, configMap)
+	if err != nil {
+		return "", err
+	}
```

Comment:
@iamkirkbater has a GET similar to this in his PR and the total account watcher has one too. Thoughts on consolidating it in a function?

Reply:
Good idea!  See https://github.com/openshift/aws-account-operator/pull/470/commits/afe82221a3668cf96250187c42ee4fba31f1124f for refactor into a function that will be more universally usable.

Should follow up for implementation with the code @iamkirkbater was working on, and the totalaccountwatcher.